### PR TITLE
fixes recyclers eating themselves roundstart

### LINKED ISSUE #59115: Recycler missing from disposals (despite being mapped in)
## Round ID:
Reproduced on local

## Reproduction:

Check disposals of any station. Notice a weird gap in the chain.

### PR DESCRIPTION
**Pull request by Kylerace**
## About The Pull Request
Fixes #59115
turfs call Entered() on all of their contents in Initialize(), since recyclers are set to recycle things that enter their tile with connect_loc, they will vore themselves. now it doesnt do that since they use LateInitialize() to register for Entered signal. 

## Why It's Good For The Game
fix

## Changelog
:cl:
fix: Recyclers dont vore themselves the moment the world is created anymore.
/:cl:

### CODE REVIEW

**Comment by Kylerace** [comment]
i caused this bug

File: code/game/machinery/recycler.dm [hunk 1]
```diff
@@ -18,20 +18,25 @@
...
+	/datum/material/glass,
+	/datum/material/silver,
+	/datum/material/plasma,
+	/datum/material/gold,
+	/datum/material/diamond,
+	/datum/material/plastic,
+	/datum/material/uranium,
+	/datum/material/bananium,
+	/datum/material/titanium,
+	/datum/material/bluespace
```
Comment (Mothblocks) [inline code/game/machinery/recycler.dm:32]: Indent one up.

Comment (Kylerace) [reply code/game/machinery/recycler.dm]: feexed

**Comment by Kylerace** [comment]
spurious test failure moment

File: code/game/machinery/recycler.dm (hunk 1 above)
Comment (Kylerace) [reply code/game/machinery/recycler.dm]: indented

**Pull request merged**